usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1742,10 +1753,15 @@
                 ("naics_code", "award__latest_transaction__contract_data__naics"),
                 ("naics_description", "award__latest_transaction__contract_data__naics_description"),
                 ("usaspending_permalink", "usaspending_permalink"),  # to be filled in by annotation
+                (
+                    "last_modified_date_NAME_CONFLICT_RESOLVER",
+                    "last_modified_date_NAME_CONFLICT_RESOLVER",
+                ),  # Column is appended to in account_download.py.  Also, this name will be replaced in post.  See
```

Comment:
Use of a variable will explain what's going on here better than a comment. 
```suggestion
                ),  
```